code/datums/status_effects/buffs.dm

```diff
@@ -188,15 +189,53 @@
 		return
 	REMOVE_TRAIT(owner, TRAIT_CHUNKYFINGERS, VAMPIRE_TRAIT)
 	var/mob/living/carbon/human/H = owner
-	H.physiology.brute_mod /= 0.5
-	H.physiology.burn_mod /= 0.8
+	H.physiology.brute_mod /= 0.4
+	H.physiology.burn_mod /= 0.5
 	H.physiology.stamina_mod /= 0.5
 	H.physiology.stun_mod /= 0.5
 	if(bonus_damage_applied)
 		bonus_damage_applied = FALSE
 		H.physiology.melee_bonus -= 10
 		H.dna.species.punchstunthreshold -= 8
 
+/datum/status_effect/vampire_gladiator
+	id = "vampire_gladiator"
+	duration = 30 SECONDS
+	tick_interval = 1 SECONDS
+	alert_type = /obj/screen/alert/status_effect/vampire_gladiator
+
+/obj/screen/alert/status_effect/vampire_gladiator
+	name = "Gladiatorial Resilience"
+	desc = "Roused by the thrill of the fight, your body has become more resistant to breaking!"
+	icon = 'icons/mob/actions/actions.dmi'
+	icon_state = "mech_damtype_brute"
+
+/datum/status_effect/vampire_gladiator/on_apply()
+	. = ..()
+	var/mob/living/carbon/human/H = owner
+	ADD_TRAIT(H, TRAIT_NO_BONES, GLADIATOR)
+	ADD_TRAIT(H, TRAIT_STURDY_LIMBS, GLADIATOR)
+	ADD_TRAIT(H, TRAIT_BURN_WOUND_IMMUNE, GLADIATOR)
+	for(var/Y in H.bodyparts)
+		var/obj/item/organ/external/E = Y
+		E.add_limb_flags()
+		E.limb_flags |= CANNOT_INT_BLEED
+
+/datum/status_effect/vampire_gladiator/on_remove()
+	var/mob/living/carbon/human/H = owner
+	REMOVE_TRAIT(H, TRAIT_NO_BONES, GLADIATOR)
+	REMOVE_TRAIT(H, TRAIT_STURDY_LIMBS, GLADIATOR)
+	REMOVE_TRAIT(H, TRAIT_BURN_WOUND_IMMUNE, GLADIATOR)
+	for(var/Y in H.bodyparts)
+		var/obj/item/organ/external/E = Y
+		E.remove_limb_flags()
```

Comment:
Please use a better variable name than `Y` here. `E` is *alright* since it's a typecast.